input/fsh/SD_Radiotherapy.fsh

```diff
@@ -215,7 +218,7 @@ Expression: "extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefini
          extension.where(url = 'http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-radiotherapy-technique').value.exists(coding.system = 'http://snomed.info/sct' and  (coding.code = 'UNC' or coding.code = '113120007'))"
 Severity: #error
 
-Invariant: TechniquesForLowDoseRateUsingTempRadSourcev1
```

Comment:
Looks like you lost an "Invariant:"

Reply:
I put it back.  Even though it is in a comment... Then I decided to delete all the comments, since they weren't properly constructed FHIRPaths.